### ISSUE TITLE
fauxectool no load.

### ISSUE THREAD

**Issue by tommar87** [DHowett/FrameworkWindowsUtils#7]
hi everything seems to work fine intill i get to step 4 with the fauxectool in cmd i run as admin i get OUT: 4|255 then nothing
![Desktop Screenshot 2024 07 11 - 18 19 59 07](https://github.com/DHowett/FrameworkWindowsUtils/assets/175352870/00fab456-4454-4844-950d-6fd6dda66c7e)

**Comment by DHowett** [comment]
You will need to reboot after enabling test signing. :)

**Comment by tommar87** [comment]
Yes I did that, I disabled secure boot did the signature stuff, restarted PC, ran installer, that did fine, entered the fauxectool and got the out: 4|255, but it doesn't populate.
